python/requirements-build.txt

```diff
@@ -1 +1,2 @@
 grpcio-tools
+dataclasses
```

Comment:
why is this needed? i do not see it imported anywhere, am i missing something?

Reply:
It's used in `structs`.  Without installing it `gen_constants` would fail.